data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -242,6 +242,27 @@
       [ "poppy_bud", 2 ]
     ]
   ]
+},{
+  "type" : "recipe",
+  "result": "bonemeal_tablet",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_DRUGS",
+  "skill_used": "cooking",
+  "skills_required": [ "survival", 2 ],
+  "difficulty": 2,
+  "time": 20000,
+  "book_learn": [["pocket_survival", 3] , ["atomic_survival", 2] , ["textbook_survival", 2] , ["survival_book", 3]],
```

Comment:
Recipe looks fine.  Increase the difficulty to 3 and add dried fruit or fruit leathers for the flavored version.